cmd/server/main.go

```diff
@@ -267,6 +271,7 @@ func newProxyRunOptions() *ProxyRunOptions {
 		clusterCaCert:          "",
 		mode:                   "grpc",
 		udsName:                "",
+		deleteUDSFile:          false,
```

Comment:
Do we want to default to true? I realize this is a change in behavior; so we need to consider backward compatibility. I believe the server won't some up if the file already exists and we are still pretty early in our adoption.

Reply:
If this flag is default to true, misconfiguring UDSName become more detrimental.

> I believe the server won't some up if the file already exists
 
That's right. I think it's better to let user find that the server doesn't come up and then manually set this flag to true, than accidentally deleting files and then disable this flag.

Reply:
Alright I'll buy that.